icons/clipboard-clock.json

```
@@ -0,0 +1,35 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "beanduong",
+    "colebemis",
+    "csandman",
+    "ericfennis",
+    "karsa-mistmere",
+    "jamiemlaw"
+  ],
+  "tags": [
+    "copy",
+    "paste",
+    "history",
+    "log",
+    "clock",
+    "time",
+    "watch",
+    "alarm",
+    "hour",
+    "minute",
+    "history",
+    "reminder",
+    "scheduled",
+    "deadline",
+    "pending",
+    "time tracking",
+    "timesheets",
+    "appointment",
+    "logbook"
+  ],
+  "categories": [
+    "text"
```

Ugh, one more thing, we do need `time` as a category, I think:

```suggestion
    "time",
    "text"
```